release-notes/2024-04/Drop-workorder-database.md

```diff
@@ -0,0 +1,28 @@
+# Dropping the work order database
+
+## Assumptions
+
+- The workorder service is enabled but not in use, no data needs to be retained.
+- SystemLink Enterprise deployment uses in-cluster, per-service mongo databases.
+- Administrator has `kubectl` access to the Kubernetes cluster from a local bash shell.
```

Comment:
@prestwick Is assuming local bash OK? Quoting would be different if it's cmd or pwsh.